test/test_dispatch.py

```diff
@@ -3,11 +3,19 @@
 from pyroapi import handlers, infer, pyro, pyro_backend, register_backend
 from pyroapi.testing import MODELS
 
+PACKAGE_NAME = {
```

Comment:
Embedding these six lines in each of two test files seems simpler than abstracting out. I intend to eventually drop test_dispatch and move its tests into test_tests.py anyway.